### PR TITLE
fix sc.type of qingcloud-csi

### PR DESCRIPTION
**Pull request by min-zh**
fix : https://github.com/kubesphere/kubesphere/issues/2883

### CODE REVIEW

**Review by zheng1 — APPROVED** [review]

File: src/test/csi-qingcloud/templates/sc.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ metadata:
     {{- end}}
 provisioner: {{ .Values.driver.name }}
 parameters:
-  {{- if kindIs "float64" .Values.sc.type }}
+  {{- if ne (.Values.sc.type | toString) "auto" }}
```
Comment (hlwanghl) [inline src/test/csi-qingcloud/templates/sc.yaml:27]: what if the value is empty `""`?

**Comment by zheng1** [comment]
Please update the image version and commit with signed off `git commit -s`.

File: src/test/csi-qingcloud/templates/sc.yaml (hunk 1 above)
Comment (min-zh) [reply src/test/csi-qingcloud/templates/sc.yaml:27]: type is “”.

**Comment by min-zh** [comment]
> Please update the image version and commit with signed off `git commit -s`.

it's already "git commit -s"

**Pull request merged**